utils/helper.js

```diff
@@ -0,0 +1,48 @@
+const Otp = require("../model/otpModel");

```

Comment:
what type of naming convention are you using, utils and helper implies the same. can you please change it?